src/zepben/evolve/model/cim/iec61968/metering/create_metering_components.py

```diff
@@ -0,0 +1,45 @@
+#  Copyright 2020 Zeppelin Bend Pty Ltd
+#
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from typing import List
+
+from zepben.evolve import *
+
+
+def create_meter(mrid: str = None, name: str = '', description: str = "", names: List[Name] = None, location: Location = None,
+                 organisation_roles: List[AssetOrganisationRole] = None, customer_mrid: str = None, service_location: Location = None, 
+                 usage_points: List[UsagePoint] = None) -> Meter:
+    """
+    Meter(EndDevice(AssetContainer(Asset(IdentifiedObject))))
+    IdentifiedObject: mrid, name, description, names
+    Asset: location, organisation_roles
+    AssetContainer:
+    EndDevice:
+    Meter:
+    """
+    m = Meter(**locals())
+    if usage_points:
+        for usage_point in usage_points:
+            usage_point.add_end_device(m)
```

Comment:
This pattern is in a lot of the creators and I feel it could be tweaked to be a little nicer by reading the assigned things back off the instance rather than using the parameter. e.g.

```
for usage_point in m.usage_points:
    usage_point.add_end_device(m)
```

@kgreav opinions?